src/ccid_usb.c

```diff
@@ -644,6 +644,39 @@ status_t OpenUSBByName(unsigned int reader_index, /*@null@*/ char *device)
 				}
 #endif
 
+#ifdef USE_COMPOSITE_AS_MULTISLOT
+				if (VENDOR_KAPELSE == GET_VENDOR(readerID))
+				{
+					switch (readerID)
+				 	{
+						case KAPELSE_KAPECV :
```

Comment:
Why to you want to handle this reader as a multislot reader if it has only 1 CCID interface?

Reply:
You are right, KAP-eCV, there is no need to handle it as a multislot reader. It is a USB composite with only one CCID interface (the first one), a HID interface and a CDC-ACM interface.